src/common.js

```diff
@@ -0,0 +1,7 @@
+
+// `TechPicks <day>/<month>/<year>` format
+const TECH_PICKS_REGEX = /^TechPicks ((0?[1-9]|[12][0-9]|3[01])\/(0[1-9]|1[012])\/((2[0-9])?[0-9]{2}))/g
```

Comment:
Don't use `g` flag as it will try to match against all possible matches in string and therefore will produce `null` the second time or needed to reset last index

> From [StackOverflow Answer](https://stackoverflow.com/a/11477474/5923666)